deployment/cloud-run/secure_api_server.py

```
@@ -22,21 +23,71 @@
+    validate_text_input,
 )
 
+# Import T5 and Whisper models
+T5_AVAILABLE = False
+WHISPER_AVAILABLE = False
+
+try:
+    from src.models.summarization.t5_summarizer import create_t5_summarizer
+    T5_AVAILABLE = True
+except ImportError as e:
+    import_logger.warning(f"T5 summarization not available: {e}")
```

Ruff (0.12.2)

Undefined name `import_logger`

(F821)